nibabel/nifti1.py

```diff
@@ -232,6 +233,17 @@
     (2003, 'rgb vector', (), "NIFTI_INTENT_RGB_VECTOR"),
     (2004, 'rgba vector', (), "NIFTI_INTENT_RGBA_VECTOR"),
     (2005, 'shape', (), "NIFTI_INTENT_SHAPE"),
+    # FSL-specific intent codes - FNIRT
```

Comment:
The FSL 5.0.9 source code ... there is a good chance that, in the near future, it will be available/linkable online via a git repository. But for the time being my reference is  http://fsl.fmrib.ox.ac.uk/fsldownloads :)